build.gradle

```diff
@@ -23,5 +23,4 @@ dependencies {
 test{
     systemProperties System.getProperties()
     useJUnitPlatform()
-}
-
+}
```

Comment:
Why am i seeing changes to this file. Please revert this.
@PranavHanwante how did you miss this